test/.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
do we need this file?

Reply:
Simplifies the development workflow for me as we are requiring a specific version of npm.